packages/browser/src/client/tester/runner.ts

```diff
@@ -53,6 +54,12 @@ export function createBrowserRunner(
       }
     }
 
+    onTaskFinished = async (task: Task) => {
+      if (this.config.browser.screenshotFailures && task.result?.state === 'fail') {
+        await page.screenshot()
```

Comment:
shouldn't check if the screenshot is available? what if running  using preview provider?

Reply:
Good catch

Reply:
Added a warning